sdk/dotnet/Pulumi/X/Automation/StackInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Pulumi.X.Automation
+{
+    public class StackInfo
+    {
+        public string Name { get; }
+
+        public bool IsCurrent { get; }
+
+        public string? LastUpdate { get; }
+
+        public bool IsUpdateInProgress { get; }
```

Comment:
I'm concerned this may be a little misleading. We don't have a way of knowing if an update is in progress (short of trying to start one) since another machine or user could be running one independently of this instance of the Stack.

Reply:
I think that is from [this.](https://github.com/pulumi/pulumi/blob/66b7794fbceb1db2c4d8d492b308d973bc167598/sdk/nodejs/x/automation/workspace.ts#L200) Should we come up with a more accurate name for that?